lib/types/addons.d.ts

```diff
@@ -0,0 +1,93 @@
+import API, { INormalizeError, IRazorpayQuery } from './api'
+import { IRazorpayItemId } from './items';
+
+
+export interface IRazorpayAddon {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏Lets remove this in favour of `IRazorpayItemCreateRequestBody`

Reply:
Done